cmdriver/cmdriver.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    prometheus.io/port: "9101"
+    prometheus.io/scrape: "true"
+  labels:
+    app: cmdriver
+  name: cmdriver
+  namespace: scaletest
+spec:
+  template:
+    metadata:
+      labels:
+        app: cmdriver
+    spec:
+      containers:
+      - name: cmdriver
+        image: marceloamaral/cmdriver:latest
+        command:
+          - "/cmdriver"
+          - "-lambda=$(LAMBDA)"
+          - "-maxpop=$(MAXPOP)"
+          - "-n=$(NUM)"
+          - "-log_dir=$(LOGDIR)"
+          - "-waitBeforeTerminate=$(WAITBEFORETERMINATE)"
+        env:
+        - name: LAMBDA
+          value: "10"
+        - name: MAXPOP
+          value: "100"
+        - name: NUM
+          value: "300"
+        - name: LOGDIR
+          value: /results
+        - name: WAITBEFORETERMINATE
+          value: "15"
+        ports:
+        - containerPort: 9101
+        volumeMounts:
+        - mountPath: /results
+          name: results
+      restartPolicy: Never
```

Comment:
This means that if/when the driver crashes, the container will not be restarted within the same Pod but rather the Pod will be destroyed and a replacement created.  I do not think we want any form of retry.  Perhaps adding `backoffLimit: 0` would accomplish that.